docs/usage/sat_bootprep.md

```diff
@@ -204,38 +204,38 @@ images:
 #### Use Base Images or Recipes from a Product
 
 Here is an example showing the definition of two images. The first image is
-built from a recipe provided by the `cos` product. The second image uses the
+built from a recipe provided by the `sma` product. The second image uses the
```

Comment:
I updated COS references in this topic to SMA since I don't think it matters what product we use. We're more concerned with the bootprep examples themselves.

Reply:
It does actually kind of matter. The COS product today provides the base image recipe that is used to build images for compute nodes and application nodes. SMA does not provide such an image, so I think this documentation will be misleading and confusing if we change it to SMA. I think we need to figure out how the image will be provided and recorded in the product catalog by the new split COS product. Then we will need to update this example accordingly. @deantroe, do you know how this example should change?

Another option would be to switch to using the barebones recipe or image provided by CSM as an example, but it's less of a real-world example because people aren't really booting that.

Reply:
Things get a little confusing starting in 23.11.  

Technically, we will be providing multiple install tar.gz files:
- COS Base - primarily the COS kernel (managed software)
- CNE for COS Base - CNE content that installs "on top of" COS Base (managed software)
- CNE for CSM - CNE content that installs on the management plane (no managed software)

From a **product team** POV, there 2 actual products with SKUs, etc.:
- CNE (which will also support RHEL and SLES in 2024)
- COS (the combination of COS Base, CNE, SLE provided by HPE)

And even here, the CNE product isn't officially launched until 2024, even though we have done the technical work to split everything out for 23.11 and support it with COS Base.

Given this, IMO we _could_ leave the documentation referring to COS, since for 23.11 we'll be dealing with COS (as the sum of CNE, COS Base, and SLE) for computes.  And I've been told that we will not support CNE + RHEL/SLES/etc ever on CSM, only "COS".

We can discuss other options of course, but this is my knee-jerk thought.

Reply:
Updating to `cne` as discussed offline in next commit.